modules/store/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I'm just fiddling with the PR, but this should have `store` as name and coverage directory.

Also as a fyi, I recently discovered we can add a name (with displayName) to the [config](https://jestjs.io/docs/en/configuration#displayname-string-object) to make it more clear which module is running - do we want to add it here? 

This would result in:

![image](https://user-images.githubusercontent.com/28659384/82581987-ab04c080-9b91-11ea-99fb-96347b2a8e14.png)

Reply:
Ahh yes, I'll update that. And yes that's a good idea